deployments/helm/icmp-responder/templates/nsc.tpl

```diff
@@ -5,7 +5,7 @@ spec:
   selector:
     matchLabels:
       networkservicemesh.io/app: "icmp-responder-nsc"
-  replicas: 4
+  replicas: 1
```

Comment:
Really depends what we want to show with this simple example. 1 replica might be enough.

Reply:
This should never have been in the PR, was a tweak I did for local development.